services/api-server/requirements/dev.txt

```diff
@@ -15,7 +15,7 @@
 --editable ../../packages/models-library
 --editable ../../packages/postgres-database
 --editable ../../packages/pytest-simcore/
---editable ../../packages/service-library[fastapi]
+--editable ../../packages/service-library
```

Comment:
are you sure about this change? you have it in dev.txt and prod.txt but not ci.txt?
And I think but I'm not sure, that you need the fastapi variant of the service-library since this is a fastapi-based service

Reply:
yes, this is wrong: all fastpi-based services should add the `[fastapi]`. Probably the requirements in _base.txt overcome in this case the requirements missing by not installing this extra

Reply:
Yep, good point. I will revert this